### DISCUSSION TITLE
algebra relazionale - 7

### DISCUSSION THREAD

**Discussion by IonutCicio** [sapienzastudentsnetwork/basi-di-dati-1#64] [Esercizi]
![image](https://github.com/sapienzastudentsnetwork/bdi2324/assets/40034429/d36e03b8-62aa-4ba2-865c-ba3c1f545331)

**Comment by daromacarlo** [comment]
![Immagine 2024-02-03 185932](https://github.com/sapienzastudentsnetwork/bdi2324/assets/135441131/ee096c42-8046-4b04-b3a1-49acbae7d63d)

**Reply by Frisbon** [reply]
Mi viene un dubbio perché se ci pensi...
Se uno studente BOB prende 27 ad A1 e 28 a P1
Nella proiezione finale avremo due record BOB, BOB e non so se é considerato un errore.

Come non detto, Ionut mi ha corretto, le istanze sono insiemi, quindi se trovo due tuple uguali una viene eliminata automaticamente.

**Comment by Frisbon** [comment]
![image](https://github.com/sapienzastudentsnetwork/bdi2324/assets/84731341/0b0854b5-fb9b-4015-921c-840b8a5c197c)
![image](https://github.com/sapienzastudentsnetwork/bdi2324/assets/84731341/89c74cb7-8bdc-46f9-bdae-627e2f90d34c)

**Comment by CiottoloMaggico** [comment]
Gli studenti che non hanno dato nessun esame non li considero del punto (b), intuitivamente non hanno una media
![image](https://github.com/sapienzastudentsnetwork/bdi2324/assets/79638284/cb6bf838-c9bf-46f4-96b3-1c3c684f1779)
